src/mapd-crossfilter.js

```diff
@@ -1500,7 +1500,7 @@ export function replaceRelative(sqlStr) {
           query +=
             " MOD(MOD(" +
             _dataTables[0] +
-            ".rowid, 4294967296) * 2654435761, 4294967296) < " +
+            ".rowid, 2147483648) * 2654435761, 4294967296) < " +
```

Comment:
Can we make this a constant, like in metis? Or at least add a comment documenting the meaning and/or origin of the magic number? If I saw that in the code, I'd be confused.